### DISCUSSION TITLE
Support CC0 1.0 license

### DISCUSSION THREAD

**Discussion by fkohrt** [quarto-dev/quarto-cli#10743] [Feature Requests]
### Description

It would be nice if the [CC0 1.0 Universal (CC0 1.0) Public Domain Dedication](http://creativecommons.org/publicdomain/zero/1.0/) would be supported as well for the `license` metadata key.

See also rstudio/distill#431

**Comment by mcanouil** [comment]
You can set whatever you want in this field.

**Reply by fkohrt** [reply]
In the documentation, [it says](https://quarto.org/docs/authoring/front-matter.html#cc):

> If you specify a Creative Commons license for your content, Quarto will automatically include the relevant link to the appropriate license.

No such mechanism exists for the CC0 license, which is also by Creative Commons.

**Reply by mcanouil** [reply]
Yes, but still you can set it manually anyway for now.

**Reply by fkohrt** [reply]
Yes, that's a great suggestion in the meantime, thanks!

**Reply by cderv** [reply]
Thanks for the idea. I'll add the support. 

- #10761

**Reply by mcanouil** [reply]
@fkohrt CC0 license will be available in the next pre-release version.
